config/galaxy.ini.sample

```diff
@@ -125,6 +125,11 @@ paste.app_factory = galaxy.web.buildapp:app_factory
 # but prefixed with install_ are also available).
 #install_database_connection = sqlite:///./database/universe.sqlite?isolation_level=IMMEDIATE
 
+# Setting the following option to true will cause Galaxy to automatically
+# migrate the database forward after updates. This is not recommended for production
+# use.
+#database_auto_migrate = False
```

Comment:
Hey @martenson - you should set database_auto_migrate to True in your galaxy.ini file - it makes development across branches easier.